wmt-rules/workload/available-points-per-offender-manager.js

```diff
@@ -0,0 +1,33 @@
+const offenderManagerTypes = require('./constants/offender-manager-types')
+const _ = require('lodash')
+
+module.exports = function (offenderManagerTypeId, contractedHours,
+        nominalTarget, hoursReduction, defaultHoursLookup) {
+  if (_.isNull(defaultHoursLookup) || _.isNull(offenderManagerTypeId) ||
+            _.isNull(nominalTarget)) {
+    return 0
+  }
+
+  if (_.includes(offenderManagerTypes.EXEMPT_IDS, offenderManagerTypeId)) {
+    return 0
+  }
+
+  var isPSO = _.includes(offenderManagerTypes.PSO_IDS, offenderManagerTypeId)
+  var defaultHours = 0
+
+  if (isPSO) {
```

Comment:
We could move the `_.includes(offenderManagerTypes.PSO_IDS, offenderManagerTypeId)` into the if statement here, if we don't need to reuse the result of the evaluation; then we could also remove the declaring of defaultHours to 0 as the else would cover it by looking up PO defaults.